src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/SetAzureRmDataLakeStoreItemAclEntry.cs

```diff
@@ -111,7 +111,7 @@ public override void ExecuteCmdlet()
                     {
                         DataLakeStoreFileSystemClient.ChangeAclRecursively(Path.TransformedPath,
                             Account,
-                            Acl.Select(entry => entry.ParseDataLakeStoreItemAce()).ToList(), RequestedAclType.ModifyAcl, Concurrency);
+                            aclSpec, RequestedAclType.ModifyAcl, Concurrency);
```

Comment:
@rahuldutta90 Please add a test to cover this change.

Reply:
@maddieclayton Added